README.md

```
@@ -42,11 +42,38 @@
+            # Method 2: Using Python's secrets module
+            python -c "import secrets; print(secrets.token_urlsafe(50))"
+            
+            # Method 3: Using openssl (if available)
+            openssl rand -base64 50
+            ```
         - `MONGODB_URI`: MongoDB connection string (default: `mongodb://localhost:27017`)
         - `DB_NAME`: Your database name
         - `GOOGLE_OAUTH_CLIENT_ID` and `GOOGLE_OAUTH_CLIENT_SECRET`: OAuth credentials for Google authentication
-        - `PRIVATE_KEY` and `PUBLIC_KEY`: Generate RSA key pairs for JWT token signing
+        - `PRIVATE_KEY` and `PUBLIC_KEY`: Generate RSA key pairs for JWT token signing using one of these methods:
+            ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)